lib/trino-filesystem-s3/src/main/java/io/trino/filesystem/s3/S3FileSystemConfig.java

```diff
@@ -23,6 +23,7 @@
 import io.airlift.units.MinDataSize;
 import jakarta.validation.constraints.Min;
 import jakarta.validation.constraints.NotNull;
+import software.amazon.awssdk.services.s3.model.ObjectCannedACL;
```

Comment:
We shouldn't bind the implementation details to the config class.

Consider rather to make the translation to `ObjectCannedACL` in `S3OutputStream`